client/src/app/site/motions/modules/motion-detail/components/motion-forward-dialog/motion-forward-dialog.component.html

```diff
@@ -1,11 +1,11 @@
 <div mat-dialog-title>{{ 'Forward motions to' | translate }}:</div>
 <div mat-dialog-content>
-    <mat-radio-group class="forward-dialog-selection" [(ngModel)]="currentSelectedMeeting">
+    <os-checkbox-group [(ngModel)]="selectedMeetings">
```

Comment:
Why do you require a custom component here? A normal list of checkboxes should suffice.

Reply:
Because I like it more, when the logic for the state management of the checkboxes is in a dedicated component.